spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java

```diff
@@ -147,21 +149,27 @@ public Object invoke(final MethodInvocation methodInvocation) throws Throwable {
                 if (globalTransactionalAnnotation != null) {
                     return handleGlobalTransaction(methodInvocation, globalTransactionalAnnotation);
                 } else if (globalLockAnnotation != null) {
-                    return handleGlobalLock(methodInvocation);
+                    return handleGlobalLock(methodInvocation, globalLockAnnotation);
                 }
             }
         }
         return methodInvocation.proceed();
     }
 
-    private Object handleGlobalLock(final MethodInvocation methodInvocation) throws Exception {
-        return globalLockTemplate.execute(() -> {
-            try {
+    private Object handleGlobalLock(final MethodInvocation methodInvocation,
+        final GlobalLock globalLockAnno) throws Throwable {
+        return globalLockTemplate.execute(new GlobalLockExecutor() {
+            @Override
+            public Object execute() throws Throwable {
```

Comment:
Why remove the try catch?

Reply:
Because the old try-catch in fact do nothing, just to wrap a throwable as a RuntimeException, so that it can match the outer method signature(throws Exception~). Here i simplify the method signature(throws Throwable), thus it can save the catch and wrap process. Just like its "brother" method(handleGlobalTransaction).

Reply:
ok